lawnchair/src/app/lawnchair/search/SearchTargetCompat.kt

```diff
@@ -262,7 +262,19 @@ data class SearchTargetCompat(
     /**
      * @hide
      */
-    @IntDef(value = [RESULT_TYPE_APPLICATION, RESULT_TYPE_SHORTCUT, RESULT_TYPE_SLICE, RESULT_TYPE_WIDGETS])
+    @IntDef(
+        value = [
+            RESULT_TYPE_APPLICATION,
+            RESULT_TYPE_SHORTCUT,
+            RESULT_TYPE_SLICE,
+            RESULT_TYPE_WIDGETS,
+            RESULT_TYPE_SUGGESTIONS,
+            RESULT_TYPE_REDIRECTION,
+            RESULT_TYPE_SECTION_HEADER,
+            RESULT_TYPE_CONTACT_TILE,
+            RESULT_TYPE_FILE_TILE,
+        ],
+    )
     @Retention(AnnotationRetention.SOURCE)
     annotation class SearchResultType
```

Comment:
We should convert this to Enum.

Reply:
I think we can do it later converting to enum in separate commit changes since it's more on refactor, or maybe you can convert it and push in this branch?

Reply:
Will do in a follow-up.